server/lib/models/model.go

```diff
@@ -95,23 +105,29 @@ func StoreColumns(dbif Dbif, record interface {}, columns []string) (int64, erro
 	}, record)
 }
 
-func (dbm *Database) Stmt(stmt string) string {
+func (dbm *Database) Stmt(stmt string, args ...interface{}) string {
```

Comment:
__[gometalinter]__
* [golint] exported method Database.Stmt should have comment or be unexported